lib/index.js

```diff
@@ -413,11 +413,11 @@ NwBuilder.prototype.copyNwjs = function () {
                 var options = {};
 
                 if(['nw', 'nwjs.app', 'nw.exe'].indexOf(destFile) !== -1){
-                    // ignore nwjs.app/Contents/Resources/*.lproj, otherwise the app name will show as nwjs.app in
+                    // ignore nwjs.app/Contents/Resources/*.lproj/InfoPlist.strings, otherwise the app name will show as nwjs.app in. *.lproj directory itself needs to be kept to support multiple locales.
```

Comment:
The word `Finder` (from the next line) should come after `nwjs.app in` (as in `... nwjs.app in Finder. *.lproj ...`)